libs/live-network/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@ledgerhq/live-network",
+  "version": "0.0.1",
```

Comment:
Version should probably be `1.0.0` right away as this version would mean we're in so early state that it could be breaking at every minor. 
I think this one is important enough for us to precise on the versionning so we know when a breaking change is really hapenning or not